src/constants/colors.ts

```diff
@@ -0,0 +1,30 @@
+const colors = {
+  primary: '#0e86d4',
+  secondary: '#003060',
+  third: '#055c9d',
+  fourth: '#68bbe3',
+  fifth: '#1f41bb',
```

Comment:
use proper names for better understanding

Reply:
I will rename them while implementing `theme` feature, please ignore them right now